include/elf.h

```diff
@@ -1602,14 +1603,15 @@ typedef struct elf64_phdr {
 } Elf64_Phdr;
 
 typedef struct elf128_phdr {
+  Elf128_Off p_offset;		/* Segment file offset */
+  Elf128_Addr p_vaddr;		/* Segment virtual address */
+  Elf128_Addr p_paddr;		/* Segment physical address */
+  Elf128_Xxword p_filesz;	/* Segment size in file */
+  Elf128_Xxword p_memsz;	/* Segment size in memory */
+  Elf128_Xxword p_align;	/* Segment alignment, file & memory */
   Elf128_Word p_type;
```

Comment:
Here also, I'd rather put p_type, p_flags and p_reserved first, so as to be "similar" to elf32/elf64 phdr.
The p_reserved here is arguably a bit strange, but it is anyway parsed in the file, so at least it makes the first part of the struct more coherent IMHO.